Prepare release v2.0.1 (#234)
1Password/connect-helm-charts: charts/connect/CHANGELOG.md

## Security
* A user-friendly description of a security fix. {issue-number}

[//]: # (START/v2.0.1)
# v2.0.1

## Fixes

> * Add `authMethod` value to set authentication method that 1Password Operator will use to access 1Password secrets. (#231)

Reviewer: ```suggestion
* Add `authMethod` value to set the authentication method used by the 1Password Operator to access 1Password secrets. (#231)
```
